grafana/Makefile

```diff
@@ -0,0 +1,25 @@
+DATASTORE_DASHBOARDS_DIR = lib/dashboards/datastore
+FRAMEWORK_DASHBOARDS_DIR = lib/dashboards/framework
+K8S_DASHBOARDS_DIR       = lib/dashboards/kubernetes
+SYSTEM_DASHBOARDS_DIR    = lib/dashboards/system
+
+.PHONY: dashboards
+dashboards:
+	curl -sSf https://grafana.com/api/dashboards/6671/revisions/2/download \
+		> $(FRAMEWORK_DASHBOARDS_DIR)/go-processes.json
+	curl -sSf https://grafana.com/api/dashboards/4701/revisions/8/download \
+		> $(FRAMEWORK_DASHBOARDS_DIR)/jvm-micrometer.json
+	curl -sSf https://grafana.com/api/dashboards/3831/revisions/1/download \
+		> $(K8S_DASHBOARDS_DIR)/kubernetes-cluster-autoscaler.json
+	curl -sSf https://grafana.com/api/dashboards/3522/revisions/1/download \
+		> $(SYSTEM_DASHBOARDS_DIR)/fluentd-elastic-logging.json
```

Comment:
Why do we need `fluentd-elastic-logging` dashboards? :thinking:

Reply:
Oops that file should not be there at all :sweat_smile: